stage1/init/init.go

```diff
@@ -185,30 +185,17 @@ func stage1() int {
 	}
 
 	mirrorLocalZoneInfo(c.Root)
-	c.MetadataServiceURL = common.MetadataServicePublicURL()
```

Comment:
@eyakubovich FWIW this broke the validator for the non private-net case:
`==> AC_METADATA_URL is not set`
Should/can this be set to some default in that case, or is it not worth it?